### DISCUSSION TITLE
No www/ directory found

### DISCUSSION THREAD

**Discussion by photoongit** [html2app/discussions#12] [Q&A]
Even with the www directory...
![Screenshot_2024-05-22-01-52-09-939_com.android.chrome-edit.jpg](https://github.com/html2app/discussions/assets/115602806/0e065b36-d3b9-47f9-9801-61f1ad2469da)

![Screenshot_2024-05-22-01-52-01-077_com.alphainventor.filemanager-edit.jpg](https://github.com/html2app/discussions/assets/115602806/9ab3d636-7998-40ba-8021-942ed44fd0c0)

**Comment by yandeu** [comment]
Can you please upload your zip file (without assets and only the index.html file in www/)?

**Reply by asaadbobby** [reply]
I am having the same problem

**Reply by yandeu** [reply]
Try the zip commands described in the template repository to zip your files:

```console
# now zip all files
(linux/macos) zip -r app.zip assets www config.json
(windows) tar.exe -acvf app.zip www assets config.json

# upload app.zip to https://html2app.dev/
```
